### PR TITLE
Portfolio - mobile menu

### PR DESCRIPTION
**Pull request by rabayarabu**
1. Add hamburger button.
2. Include the close button
3. Add menu options with the correct part of the page displayed

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -10,12 +10,43 @@ body {
...
+  display: flex;
+  flex-direction: column;
+  position: absolute;
+  gap: 20px;
+  left: 0;
+  top: 0;
+  width: 100%;
+  padding: 100px;
+  backdrop-filter: blur(8px);
+}
```
Comment (levy002) [inline style.css]: - [ ] Kindly make sure when the mobile menu is opened it fits the whole height visible screen and make sure to give the container of the menu the background image in order to match the provided Figma design as much as possible.

- [ ] KIndly make sure the navigation links have the same typography( `font, colors, font-size, font-weight, etc...`) as the provided design. Please make sure each nav link has an underline style.


   - ### YOUR DESIGN
   
    image

   - ### FIGMA DESIGN
   
    image

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @rabayarabu  , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter error :heavy_check_mark: 

## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 2]
```diff
@@ -10,12 +10,72 @@ body {
...
+
+.menu_link {
+  border-bottom: 2px solid #f4f5f7;
+  display: block;
+  padding-bottom: 5px;
+  font-weight: 700;
+  font-size: 2rem;
+  color: #fff5e1;
+}
```
Comment (Qoosim) [inline style.css]: - [ ] Great job implementing changes by the previous reviewer. However, there're few things you still need to work on.
- First of them, you're required to stick to the Figma design as close as possible. The color for the border-bottom you used is white as opposed to the color on the figma design. The hex code for the color used on the figma is this :point_right: `background: #6F6C6B;`. Kindly work on that.
- Secondly, the width of the border-bottom is extended down to the under `X` icon on the figma design as opposed to how it is on your design. Kindly look into that too and fix.

### Your Design
your-menu

### Figma Design
temp-menu

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle: 

Hi @rabayarabu, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 



 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Gambit142 — APPROVED** [review]
Hi @rabayarabu @Megagig @katfogy 👋,

## Status: Approved 🟢   

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

### Highlights
- All linters checks passed ✔️ 
- Use of JavaScript to make mobile menu functional ✔️ 
- Each navigation link takes users to the correct part of the page ✔️ 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**